linter/main.go

```diff
@@ -438,7 +432,7 @@ func main() {
 
 func arrayStringContains(inputArray []string, str string) bool {
 	for _, v := range inputArray {
-		if strings.Contains(v, str) {
+		if regexp.MustCompile(v).MatchString(str) {
```

Comment:
Wouldn't `MatchString` fail if it's not an exact match? For example, there's no job with the exact name `eks-distro-base-tooling-presubmit` but several job names contain that text.

Reply:
yeah, `MatchString` matches substrings -- check it here https://go.dev/play/p/KJNbnob95gT